vacasa/connect/connect.py

```diff
@@ -523,3 +523,36 @@ def get_reservations(self, params: dict = None):
         headers = self._headers()
 
         return self._iterate_pages(url, headers, params)
+
+    def get_quote_by_id(self, unit_id: int, arrival_date: str, departure_date: str, adult_count: int, params: dict = None):
```

Comment:
Mind adding a line break here? Since this is public I'd love to see it conform to the PEP8 style guide for maximum line length.